### ISSUE TITLE
When SLI is enabled on the Windows Host, only 1 GPU shows up in WSL

### ISSUE THREAD

**Issue by d-colwell** [microsoft/WSL#5484]
I know this is probably not a major issue, but i am running 2x Titan RTX for ML, and when i tried to run the training loop, i could only see 1 GPU.
Disabling SLI in NVidia Control panel and rebooting the WSL instance resolved the situation.

**Comment by XiaomoWu** [comment]
@d-colwell Man, you saved my day. I've been beating my head against the wall for days by the issue. At least I can now exploit the full capacity of multi-gpus. I'm not a hard-core game player so the loss of SLI is acceptable. But I still expect MSFT or Nvidia could fix this issue.

I filed a similar issue on Nvidia's WSL2 forum: [Only one GPU is recognized in WSL2](https://forums.developer.nvidia.com/t/only-one-gpu-is-recognized-in-wsl2/129526)
